### PR TITLE
Platform update

### PR DESCRIPTION
**Pull request by patricia-ternes**
This PR add to the Research Platforms page:

- Roadmap
- A more detailed list of resources

All the content was drafted in a meeting with @jhodrien;

It is the first draft of the content, as more and more details can be added, but I don't want to spend days going in one direction if it is not the right one.

The tables are in `html` because I want to include bullet points in the cells; the wordpress is in html as well, so I didn't bother to find a different solution.

### CODE REVIEW

File: pages/research_platforms.md [hunk 1]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+   - These shared systems give you extra memory, CPU power, or GPU access without the complexity of an HPC queue. Great for testing and debugging.
+3. **Cloud Solutions for Flexibility and Collaboration**
+   - For collaborative projects, cloud resources offer flexible environments.
+   - These are perfect for sharing work, developing in teams, and quickly testing your code without investing in hardware.
+   - Unfortunately, most of the solutions are not supported by the university
+4. **Need to Scale Up? Local HPC is the Next Step**
+   - When your research requires heavy computation, high-end GPUs, or parallel processing, it’s time to consider our local (Tier 3) HPC resources like.
+   - Get direct support from the Research Computing team and access systems tailored to your project’s needs.
+5. **For the Largest Projects: External HPC**
+   - For large, multi-node, or specialized GPU jobs, Tier-2 (Regional) and Tier-1 (National) HPC facilities offer matchless power and scale.
```
Comment (jhodrien) [inline pages/research_platforms.md:37]:
```suggestion
   - For large, multi-node, or specialized GPU jobs, Tier-2 (Regional) and Tier-1 (National) HPC facilities offer unmatched power and scale.
```

Comment (jhodrien) [reply pages/research_platforms.md:37]: Also, you put "Tier 3" higher up but "Tier-2" here.  Pick on or the other style.

File: pages/research_platforms.md [hunk 2]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+      <th>Platform</th>
+      <th>Application</th>
+      <th>Information</th>
+  </thead>
+  <tbody>
+    <!-- New row -->
+    <tr>
+      <td>Standard Laptops</td>
+      <td><ul>
+        <li>Standard request/support via University IT - faster deliver/support</li>
```
Comment (jhodrien) [inline pages/research_platforms.md:57]:
```suggestion
        <li>Standard request via University IT - fast delivery/setup</li>
```

File: pages/research_platforms.md [hunk 3]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+        <li>Flexible and Portable</li>
+        <li>Impacts your other work whilst running computationally intensive jobs</li>
+      </ul></td>
+      <td><a href="https://it.leeds.ac.uk/it?id=sc_cat_item&sys_id=37351d301bd55110f23fa8a4bd4bcbb5" target="_blank">Request Form</a></td>
+    </tr>
+    <!-- New row -->
+    <tr>
+      <td>Standard Windows Desktops</td>
+      <td><ul>
+        <li>Standard request/support via University IT - faster deliver/support</li>
```
Comment (jhodrien) [inline pages/research_platforms.md:67]:
```suggestion
        <li>Standard request via University IT - fast delivery/setup</li>
```

File: pages/research_platforms.md [hunk 4]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+        <li>Impacts your other work whilst running computationally intensive jobs</li>
+        <li>Not necessarily considerably faster than laptops</li>
+      </ul></td>
+      <td><a href="https://it.leeds.ac.uk/it?id=kb_article&sys_id=9e36085b1bcf02d0d5d60e51f54bcb92" target="_blank">Request Form</a></td>
+    </tr>
+    <!-- New row -->
+    <tr>
+      <td>Non-Standard Computer</td>
+      <td><ul>
+        <li>Request/support via University IT</li>
```
Comment (jhodrien) [inline pages/research_platforms.md:77]:
```suggestion
        <li>Request via University IT</li>
```

File: pages/research_platforms.md (hunk 2 above)
Comment (jhodrien) [reply pages/research_platforms.md:57]: I think stressing the support side here is potentially confusing, as these are supported by IT, but so is Windows Desktop, and various other bits too like the HPC.

File: pages/research_platforms.md [hunk 5]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+      <th>Platform</th>
+      <th>Application</th>
+      <th>Information</th>
+  </thead>
+  <tbody>
+    <!-- New row -->
+    <tr>
+      <td>Clusters</td>
+      <td><ul>
+        <li>Useful for trialing and debugging on a high end machine without needing to buy a workstation or queue on the HPC for testing</li>
```
Comment (jhodrien) [inline pages/research_platforms.md:110]:
```suggestion
        <li>Useful for trialling and debugging on a high end machine without needing to buy a workstation or queue on the HPC for testing</li>
```

File: pages/research_platforms.md [hunk 6]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+        <li><a href="https://it.leeds.ac.uk/it?id=clusters" target="_blank">Overview</a></li>
+        <li><a href="https://it.leeds.ac.uk/it?id=nr_software_a_z" target="_blank">Software List</a></li>
+        <li><a href="https://it.leeds.ac.uk/it?id=kb_article&sysparm_article=KB0011754" target="_blank">Cluster Map</a></li>
+      </ul></td>
+    </tr>
+    <!-- New row -->
+    <tr>
+      <td>Linux Workstations/Servers</td>
+      <td><ul>
+        <li>Benefits of RAM/Cores/GPU, a general issue of scale compared to personal systems</li>
```
Comment (jhodrien) [inline pages/research_platforms.md:126]:
```suggestion
        <li>Benefits of higher specification RAM/cores/GPU; generally high end compared to personal systems</li>
```

File: pages/research_platforms.md [hunk 7]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+  <thead>
+      <th>Platform</th>
+      <th>Application</th>
+      <th>Features</th>
+      <th>Information</th>
+  </thead>
+  <tbody>
+    <!-- New row -->
+    <tr>
+      <td>Colab</td>
```
Comment (jhodrien) [inline pages/research_platforms.md:146]:
```suggestion
      <td>Google Colab</td>
```

File: pages/research_platforms.md [hunk 8]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+      <td>Jupyter Notebook service useful for teaching, training and development</td>
+      <td><ul>
+        <li>Can be useful for trialling and experimenting with GPU Python codes</li>
+        <li>Relatively limited CPU (2 vCPU, 13G RAM)</li>
+        <li>No control about resource allocation - bad for reproducibility</li>
+      </ul></td>
+      <td><a href="https://colab.research.google.com/" target="_blank">Colab Homepage</a></td>
```
Comment (jhodrien) [inline pages/research_platforms.md:153]:
```suggestion
      <td><a href="https://colab.research.google.com/" target="_blank">Google Colab Homepage</a></td>
```

File: pages/research_platforms.md [hunk 9]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+    </tr>
+    <!-- New row -->
+    <tr>
+      <td>MyBinder</td>
+      <td>Can install dependencies more nicely than Colab</td>
```
Comment (jhodrien) [inline pages/research_platforms.md:158]:
```suggestion
      <td>Easier to install dependencies than Colab</td>
```

File: pages/research_platforms.md [hunk 10]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+        <li><a href="https://it.leeds.ac.uk/it?id=kb_article_view&table=kb_knowledge&sys_kb_id=b99bdf791ba74298e0502060f54bcbcd" target="_blank">IT GH Enterprise Article</a></li>
+      </ul></td>
+    </tr>
+    <!-- New row -->
+    <tr>
+      <td>GitHub actions</td>
+      <td>A model for well defined workflows. CI/CD lifecycle</td>
+      <td><ul>
+        <li>Automating code testing and deployment</li>
+        <li>Run actions in windows, linux and macos environments</li>
```
Comment (jhodrien) [inline pages/research_platforms.md:187]:
```suggestion
        <li>Run actions in Windows, Linux and macOS environments</li>
```

File: pages/research_platforms.md [hunk 11]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+      <th>Features</th>
+      <th>Information</th>
+  </thead>
+  <tbody>
+    <!-- New row -->
+    <tr>
+      <td>ARC3/4†</td>
+      <td>Higher RAM/core/storage than can be achieved in previous options</td>
+      <td><ul>
+        <li>GPUs (V100), but a relatively small number</li>
```
Comment (jhodrien) [inline pages/research_platforms.md:214]:
```suggestion
        <li>GPUs (Nvidia Tesla V100), but a relatively small number</li>
```

File: pages/research_platforms.md [hunk 12]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+      </ul></td>
+      <td><ul>
+        <li><a href="https://it.leeds.ac.uk/it?id=sc_cat_item&table=sc_cat_item&sys_id=4c002dd70f235f00a82247ece1050ebc" target="_blank">Request an Account</a></li>
+        <li><a href="https://arcdocs.leeds.ac.uk/welcome.html" target="_blank">ARC Documentation</a></li>
+      </ul></td>
+    </tr>
+    <!-- New row -->
+    <tr>
+      <td>Aire‡</td>
+      <td>11 times more flops than ARC3 and ARC4 combined</td>
```
Comment (jhodrien) [inline pages/research_platforms.md:226]:
```suggestion
      <td>11 times more FLOPS than ARC3 and ARC4 combined</td>
```

File: pages/research_platforms.md [hunk 13]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+        <li>Login nodes have GPUs, so useful for debugging and development</li>
+      </ul></td>
+      <td><a href="https://n8cir.org.uk/bede/" target="_blank">Bede Overview</a></td>
+    </tr>
+    <!-- New row -->
+    <tr>
+      <td>JADE2†</td>
+      <td>Running multiple GPU jobs - simpler than Bede due to the x86\_64 architecture</td>
+      <td><ul>
+        <li>GPU, V100</li>
```
Comment (jhodrien) [inline pages/research_platforms.md:268]:
```suggestion
        <li>100% GPU, Nvidia Tesla V100</li>
```

File: pages/research_platforms.md [hunk 14]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
-| Research Cloud                                                  | Access to Public Cloud services for research needs not met elsewhere (eg. VMs with GPUs or FPGAs, Cognitive Services APIs, Kubernetes, Azure Labs, Machine Learning Workbench).                                                                                                                                                                                                   |
-| Research Visualisation Platform                                 | Providing access to a variety of computationally intense visualisation tools (eg. VMD, Paraview).                                                                                                                                                                                                                                                                                 |
+† *The JADE2 account registration period closed on 01/09/2024. As of this date, we are no longer able to approve project requests. 6th January 2025: All access to the service will be withdrawn and physical decommissioning of the system will commence.*
```
Comment (jhodrien) [inline pages/research_platforms.md:298]:
```suggestion
† *The JADE2 account registration period closed on 01/09/2024. As of this date, we are no longer able to approve project requests. 1st November 2024: Batch and interactive access to all compute resources will be withdrawn. 6th January 2025: All access to the service will be withdrawn and physical decommissioning of the system will commence.*
```

**Review by jhodrien — CHANGES_REQUESTED** [review]
All in, I think this is much more useful to people than what we had before, and it feels easy to add to, or link other materials.  Thanks.  Suggested changes are very minor, please accept or ignore to taste.

If I was to suggest a future addition, I think it's some sort of computational consideration flowchart, so that a new PhD student can work through a set of questions as to what they're trying to do, and find the likely best platform to use.

File: pages/research_platforms.md [hunk 15]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+        <li>Total of 750k cores</li>
+      </ul></td>
+      <td><a href="https://www.archer2.ac.uk/about/hardware.html" target="_blank">ARCHER2 Overview</a></td>
+    </tr>
+    <!-- New row -->
+    <tr>
+      <td>Jasmin</td>
+      <td>A much more comprehensive service for NERC users</td>
+      <td><ul>
+        <li>Jupyer notebook support</li>
```
Comment (jhodrien) [inline pages/research_platforms.md:288]:
```suggestion
        <li>Jupyter notebook support</li>
```

File: pages/research_platforms.md [hunk 16]
```diff
@@ -3,16 +3,296 @@ title: Research Platforms
...
+   - Ideal for initial code development, small data analysis, and non-intensive simulations.
+2. **Need More Power? Try Shared Devices**
+   - If your work is starting to slow down your individual device, consider using a cluster machine or Linux server.
+   - These shared systems give you extra memory, CPU power, or GPU access without the complexity of an HPC queue. Great for testing and debugging.
+3. **Cloud Solutions for Flexibility and Collaboration**
+   - For collaborative projects, cloud resources offer flexible environments.
+   - These are perfect for sharing work, developing in teams, and quickly testing your code without investing in hardware.
+   - Unfortunately, most of the solutions are not supported by the university
+4. **Need to Scale Up? Local HPC is the Next Step**
+   - When your research requires heavy computation, high-end GPUs, or parallel processing, it’s time to consider our local (Tier 3) HPC resources like.
```
Comment (patricia-ternes) [inline pages/research_platforms.md:34]:
```suggestion
   - When your research requires heavy computation, high-end GPUs, or parallel processing, it’s time to consider our local (Tier 3) HPC resources.
```

[the remark this answers is not in the document]
Comment (patricia-ternes) [reply pages/research_platforms.md]: No, I will include the options just in the table to not repeat to much information.

**Comment by patricia-ternes** [comment]
@jhodrien everything updated, can you approve it if you are happy?

**Review by jhodrien — APPROVED** [review]

**Pull request merged**